src/components/error-notice.ts

```diff
@@ -0,0 +1,37 @@
+import { LitElement, css, html } from "lit";
+import { customElement } from "lit/decorators.js";
+
+@customElement("error-notice")
+export class ErrorNotice extends LitElement {
+  render() {
+    return html`
+      <div class="notice error">
+        <slot></slot>
+      </div>
+    `;
+  }
+
+  static styles = css`
+    :host {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      max-width: 450px;
```

Comment:
it's a bit weird there is a max width on a `:host` selector. It limits reusability. That should be added to the parent.